engine/src/main/java/org/terasology/logic/console/commandSystem/MethodCommand.java

```diff
@@ -89,17 +89,20 @@ public static void registerAvailable(Object provider, Console console, Context c
         for (Method method : commandMethods) {
             final Class[] paramTypes = method.getParameterTypes();
             final Annotation[][] paramAnnotations = method.getParameterAnnotations();
+            boolean hasSenderAnnotation = false;
             for (int i = 0; i < paramAnnotations.length; i++) {
                 if(paramTypes[i].getTypeName().equals(ENTITY_REF_NAME)) {
                     if(paramAnnotations[i].length == 0) {
-                        logger.info("Command {} contains a EntityRef without @Sender annotation, may cause a NullPointerException", method.getName());
+                        logger.error("Command {} provided by {} contains a EntityRef without @Sender annotation, may cause a NullPointerException", method.getName(), provider.getClass().getSimpleName());
```

Comment:
You could replace this line with `hasSenderAnnotation=false`.
Then you can extract Line 90 to 102 to a method `hasSenderAnnotation0.`
Inside this method the `hasSenderAnnotation` can be replaced with return statements.

Then this should be good to go 👍